lib/lita_versioner/handlers/versioner.rb

```diff
@@ -1,12 +1,22 @@
+require "lita"
 require "json"
 require "uri"
-require_relative "../project_repo"
+require_relative "../../lita_versioner/project_repo"
 require_relative "bumpbot_handler"
 
 module Lita
   module Handlers
     class Versioner < BumpbotHandler
 
+      # Common configuration must be defined once
```

Comment:
Why this?  Isn't this all defined in the base class?  Now we'd have 2 sets of config options.